src/customizations/volto/components/manage/Form/BlocksToolbar.jsx

```diff
@@ -17,7 +17,7 @@ import { Icon } from '@plone/volto/components';
 import { Plug } from '@plone/volto/components/manage/Pluggable';
 import { load } from 'redux-localstorage-simple';
 import { isEqual, omit, without } from 'lodash';
-import { cloneBlock } from 'design-comuni-plone-theme/config/Blocks/ListingOptions';
```

Comment:
Mi aspetterei che venga cancellata la funzione `cloneBlock` dal file in cui era importata prima

Reply:
Giusto, avevo pensato di farlo infatti, ma avevo una riunione con Nicola e dopo mi sono scordato. Grazie!

Reply:
Ho anche cambiato i path per le altre import, principqlmente dentro al blocco listing